index.js

```diff
@@ -531,6 +531,14 @@ function Argv (processArgs, cwd) {
     return argv
   }
 
+  function guessLocale () {
+    try {
+      return osLocale.sync()
```

Comment:
It only spawns when neither of the following environment variables are defined:

```
env.LC_ALL || env.LANGUAGE || env.LANG || env.LC_MESSAGES
```

Might be a good idea to add an option to `os-locale` to prevent spawning `{spawn: false}`?